sdk/schemaregistry/Azure.Data.SchemaRegistry/tests/Samples/Sample01_ReadmeSnippets.cs

```diff
@@ -25,7 +25,7 @@ public void CreateSchemaRegistryClient()
             // Create a new SchemaRegistry client using the default credential from Azure.Identity using environment variables previously set,
             // including AZURE_CLIENT_ID, AZURE_CLIENT_SECRET, and AZURE_TENANT_ID.
             // For more information on Azure.Identity usage, see: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/identity/Azure.Identity/README.md
-            var client = new SchemaRegistryClient(endpoint: endpoint, credential: new DefaultAzureCredential());
+            var client = new SchemaRegistryClient(fullyQualifiedNamespace: endpoint, credential: new DefaultAzureCredential());
```

Comment:
Should we update `endpoint` here to `fullyQualifiedNamespace` to match the pattern in the other README?